### ISSUE TITLE
Update Timings based on measured times

### ISSUE THREAD

**Issue by bpmweel** [carpentries-incubator/deep-learning-intro#124]
Update the episode timings based on what we measured during the june workshop.

**Comment by bpmweel** [comment]
Here are the timings we measured when we taught it:

### Day 1
- 9:30 Started episode 1
- 10:35 Break 1 : check installation. Finished exercise 1 and explanation about deep learning, not yet deep learning workflow
  Couldn't resume after 15 min because fixing setups took more time.
  Breakout Exercise  Deep learning workflow needs a bit more than 10 min (15 min seems OK)
- 11:10 Finish episode 1
- 11:37 Break 2: just finished pairplotting exercise
- 12:50 Finish:  finished Exercise: Create the neural network (silent docing)

### Day 2
-	9:15 start recap + Q&A yesterday
-	9:35 start choosing a cost function
-	9:51 start training history exercise
-	10:00 finish training history exercise
-	10:22 Confusion matrix exercise
-	10:31 Confusion matrix exercise stops, start break
-	10:53 Stop break, Start weather prediction
-	11:05 Start data inspection exercise weather prediction
-	11:?? Stop exercise
-	11:30 Start exercise train/test/val set
-	11:40 Stop exercise
-	11:47 Start break
-	12:05 Start ‘build a neural network’ exercise
-	12:22 Finish exercise
-	13:00 Finished training the model and inspecting the evaluation on train and test set

### Day 3
•	9:19 start recap yesterday
•	9:31 start exercise create a similar scatterplot for a reasonable baseline
•	9:45 ?? Stop exercise
•	10:07 start exercise plotting validation history
•	10:12?? Stop exercise
•	10:20 Start exercise less parameters in breakout rooms
•	10:30 Stop exercise less parameters
•	10:50 Start explaining early stopping
•	11:05 Start exercise batchnorm
•	11:20?? Stop exercise batchnorm
•	11:30 Start exercise dropout
•	11:48 Stop exercise dropout
•	BREAK
•	12:07: Discuss dropout exercise
•	12:12 saving and loading model (start)
•	12:20??: start exercise on how to further improve the model
•	12:35??: stop exercise
•	12:50 start survey

**Comment by bpmweel** [comment]
Fixed with #126
